### PR TITLE
Add travis build file

### PR DESCRIPTION
**Pull request by jvgelder**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+dist: trusty
```
Comment (gertvdijk) [inline .travis.yml]: bionic or focal please?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
```
Comment (gertvdijk) [inline .travis.yml:6]: why R testing toolchain? seems rather random

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    packages:
+      - wget
+      - pkg-config
+
+before_install:
+  - wget https://github.com/bazelbuild/bazelisk/releases/download/v1.6.0/bazelisk-linux-amd64
+  - echo "616f65bcdcfd134a19d5d86c591c35098d7732be26145bf06f02ec9e3e52700c bazelisk-linux-amd64" | sha256sum --check
+  - sudo mv bazelisk-linux-amd64 /usr/bin/bazelisk
```
Comment (gertvdijk) [inline .travis.yml]: copy to local path relative to working directory, then modify path. e.g. like https://docs.travis-ci.com/user/installing-dependencies/#installing-projects-from-source

Comment (jvgelder) [reply .travis.yml:6]: I hope you don't mind my current solution :)

**Pull request closed without merging**